tools/Customization/DevHome.Customization/ViewModels/DevDriveInsightsViewModel.cs

```diff
@@ -307,12 +372,20 @@ public void UpdateOptimizerListViewModelList()
                 continue;
             }
 
+            List<string> existingDevDriveLetters = new List<string>();
+            foreach (var existingDevDrive in ExistingDevDrives)
+            {
+                existingDevDriveLetters.Add(existingDevDrive.DriveLetter.ToString());
+            }
+
+            var exampleDirectory = Path.Join(existingDevDriveLetters[0] + ":", cache.ExampleSubDirectory);

```

Comment:
You sure that existingDevDriveLetters[0] will never be empty?

Reply:
Yes, this code never executes if there are no dev drives. We do not display Dev Drive Insights card at all.